robot_parameters:
    starting_mode: 1
    J1:
        ROM_Min: 0.0
        ROM_Max: 0.0

    J2:
        ROM_Min: 0.0
        ROM_Max: 0.0

    J3:
        ROM_Min: 0.0
        ROM_Max: 0.0

user_parameters:
    weight: 70.0
    height: 178.0
    forearm_length: 25.0
    upperarm_length: 25.0
